src/ai/backend/manager/models/session.py

```diff
@@ -1309,7 +1309,7 @@ async def _get_and_transit(
             transited = session_row.determine_and_set_status(status_changed_at=now)
 
             def _calculate_session_occupied_slots(session_row: SessionRow):
-                session_occupying_slots = ResourceSlot.from_json({**session_row.occupying_slots})
+                session_occupying_slots = ResourceSlot()
```

Comment:
The current code seems a bit tricky to write test code... 😢 
Have you tried verifying that it works?

Reply:
yeah I tested it